public/Player.js

```diff
@@ -32,22 +36,96 @@ export default class Player {
     this.domElement = htmlPlayersContainer.querySelector(`#${this.name}`);
   }
 
+  sumOfCards() {
+    let sumOfHand = 0;
+
+    const [arrOfCardsWithFixedValues, arrOfCardsWithAlternateValues] =
+      this.hand.reduce(
+        ([arrOfCardsWithFixedValues, arrOfCardsWithAlternateValues], card) => {
+          if (card.alternateValue) {
+            arrOfCardsWithAlternateValues.push(card);
+          } else {
+            arrOfCardsWithFixedValues.push(card);
+          }
+          return [arrOfCardsWithFixedValues, arrOfCardsWithAlternateValues];
+        },
+        [[], []]
+      );
+
+    let amountOfAces = arrOfCardsWithAlternateValues.length;
+
+    for (let card of arrOfCardsWithFixedValues) {
+      if (card.faceDirection === "down") break;
+      sumOfHand += card.value;
+    }
+
+    if (amountOfAces !== 0) {
+      for (let card of arrOfCardsWithAlternateValues) {
+        if (card.faceDirection === "down") break;
+        if (sumOfHand + card.value > 21) {
+          sumOfHand += card.alternateValue;
+        } else {
+          sumOfHand += card.value;
+        }
+      }
+    }
+
+    this.handValue = sumOfHand;
+    return;
+  }
+
+  renderHandValue() {
+    const sumContainer = document
+      .querySelector(`#${this.name}`)
+      .querySelector(".player__hand-value");
+    sumContainer.innerHTML = this.handValue.toString();
+  }
+
   receiveCard(card) {
     this.hand.push(card);
     this.renderNextCard();
+    this.sumOfCards();
+    this.renderHandValue();
   }
 
   renderNextCard() {
     if (this.nextCardToRender > this.hand.length - 1) return;
     let htmlString = "";
     const cardToRender = this.hand[this.nextCardToRender];
     if (cardToRender.faceDirection === "down") {
-      htmlString = "Card down";
+      htmlString = '<p id="hidden-card">Card down</p>';
     } else {
       htmlString += `<p>${cardToRender.number} of ${cardToRender.suit} <i class="${cardToRender.icon}  "></i></p>`;
     }
     const cardContainerElem = this.domElement.querySelector(".card");
     cardContainerElem.insertAdjacentHTML("afterbegin", htmlString);
     this.nextCardToRender++;
   }
+
+  secondDealerCard() {
+    const secondCard = this.hand.pop();
+    secondCard.flipCard();
+    document.querySelector("#hidden-card").remove();
+    return secondCard;
+  }
+
+  addFocurs() {
```

Comment:
There is a typo here:
```suggestion
  addFocus() {
```